### DISCUSSION TITLE
Kanban view

### DISCUSSION THREAD

**Discussion by Himlol** [TriliumNext/Trilium#3474] [Q&A] [answered]
Hi guys,

Has anyone already made a minimal working version of the kanban board view with drag&drop for this world's best note taking app? 
I have seen several issues ([#690](https://github.com/zadam/trilium/issues/690), [#1402](https://github.com/zadam/trilium/issues/1402), [1412](https://github.com/zadam/trilium/issues/1412)) and amazing extension [Collection Views](https://github.com/mabeyj/trilium-collection-views).
But there doesn't seem to be anything that could be tried (and improved or something).

**Comment by kooshi** [comment]
I'm going down this rabbit hole as well.  #2479 is about this as well, and @rebron1900 seems like he was making progress, but I don't see it in his repos. @Tukks also has a forked simple vanilla js kanban board in his repos, which would have been a step toward the goal, but I don't see a finished Trilium one.

I'll keep looking, and if I get fed up enough, maybe I'll try to make one, but in the mean time, I'll keep using [Vikunja](https://vikunja.io) along side Trilium.

Thanks for linking collection views though. I hadn't found it yet. Trilium needs an `awesome-trilium-scripts` repo.

**Comment by CyrilLeblanc** [comment]
Hey, I made a custom implementation of this feature using JKanban.
If you are interested here's the repo :
https://github.com/CyrilLeblanc/trilium-kanban

**Comment by laundmo** [comment, marked as answer]
Kanban is now integrated into trilium with the note collection of type Board
